extensions/registries/src/extension.ts

```diff
@@ -58,16 +53,11 @@ const defaultRegistries: extensionApi.RegistrySuggestedProvider[] = [
   {
     name: 'GitHub',
     url: 'ghcr.io',
-    icon: ghcrIoImage,
+    icon: { light: ghcrIoImageDark, dark: ghcrIoImageLight },
   },
   {
     name: 'Google Container Registry',
     url: 'gcr.io',
     icon: gcrIoImage,
   },
 ];
-
-// Remove data:image/png;base64,
-export function stripImagePrefix(imageContent: string): string {
```

Comment:
some tests seems to be failing since this has been removed?

Reply:
right, thanks, method was removed, but the following test wasn't, fixed in the following commit